src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -760,6 +781,12 @@ public NumericMapping numericMapping() {
     return NumericMapping.get(this);
   }
 
+  public TimeZone timeZone() {
+    String dbTimeZone = getString(JdbcSourceTaskConfig.DB_TIMEZONE_CONFIG);
+    return dbTimeZone == null ? TimeZone.getTimeZone(ZoneOffset.UTC) :
+        TimeZone.getTimeZone(ZoneId.of(dbTimeZone));
```

Comment:
Same comment as above: the default plus a validator means that we should always have a valid `dbTimeZone` string that will always correspond to a valid time zone, and the code could be simplified to:

    String dbTimeZone = getString(JdbcSourceTaskConfig.DB_TIMEZONE_CONFIG);
    return TimeZone.getTimeZone(ZoneId.of(dbTimeZone));

Reply:
:+1: